github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -1881,6 +1881,32 @@ presubmits:
           privileged: true
       nodeSelector:
         type: bare-metal-external
+  - always_run: true
+    annotations:
+      fork-per-release: "true"
+    cluster: prow-workloads
+    decorate: true
+    decoration_config:
+      grace_period: 5m0s
+      timeout: 1h0m0s
+    max_concurrency: 11
+    name: pull-kubevirt-code-lint
+    skip_branches:
+    - release-\d+\.\d+
+    spec:
+      containers:
+      - command:
+        - /usr/local/bin/runner.sh
+        - /bin/sh
+        - -c
+        - make lint
```

Comment:
@rhrazdil I think this line won't work because the `lint` word would be cut. Please use the syntax like in [this](https://github.com/kubevirt/project-infra/blob/d7cf80b4872a241a7bffca05e1f018249c08ad6d/github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml#L1838) line

Reply:
You're right, thanks! [Fixed](https://github.com/kubevirt/project-infra/compare/498d2712f9cba2c5ea9647c5cd468cd14f3b0ded..83308dc3a07b358862c995faa9e6ec6201879276)